UE4SS/include/SDKGenerator/UEHeaderGenerator.hpp

```diff
@@ -91,21 +93,21 @@ namespace RC::UEGenerator
 
     struct StringInsensitiveCompare
     {
-        auto operator()(const std::wstring& a, const std::wstring& b) const -> bool
+        auto operator()(const StringType& a, const StringType& b) const -> bool
         {
-            return _wcsicmp(a.c_str(), b.c_str()) < 0;
+            return RC::Unreal::FGenericPlatformStricmp::Stricmp((RC::Unreal::UTF16CHAR*) a.c_str(), (RC::Unreal::UTF16CHAR*)b .c_str()) < 0;
```

Comment:
Some weird formatting here.

Reply:
Doesn't look like weird formatting to me, but it does use two C-casts which is bad, an I don't think it's even correct ?
The `Stricmp` function has overloads for all the string types so I'm not sure why we need to cast or convert them.
I will remove these casts, and someone will have to test that UHT is still working properly after this.